src/classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  attr_accessor :label, :students
+
+  def add_student(student)
+    @students << student unless @students.include?(student)
+    student.update_classroom(self) unless student.classroom == self
```

Comment:
👍👍👍👍 Great use of defensive code to ensure that an entry isn't made more than once.